app/src/main/java/com/kickstarter/libs/KSCurrency.java

```diff
@@ -157,27 +190,76 @@ private String getSymbolForCurrency(final String chosenCurrency) {
     } else if (chosenCurrency.equals(CurrencyCode.GBP.rawValue())) {
       return "£";
     } else if (chosenCurrency.equals(CurrencyCode.HKD.rawValue())) {
-      return "HK$";
+      return "HK$ ";
     } else if (chosenCurrency.equals(CurrencyCode.JPY.rawValue())) {
       return "¥";
     } else if (chosenCurrency.equals(CurrencyCode.MXN.rawValue())) {
-      return "MX$";
+      return "MX$ ";
     } else if (chosenCurrency.equals(CurrencyCode.NOK.rawValue())) {
       return "NOK";
     } else if (chosenCurrency.equals(CurrencyCode.NZD.rawValue())) {
-      return "NZ$";
+      return "NZ$ ";
     } else if (chosenCurrency.equals(CurrencyCode.SEK.rawValue())) {
       return "SEK";
     } else if (chosenCurrency.equals(CurrencyCode.SGD.rawValue())) {
-      return "S$";
+      return "S$ ";
     } else if (chosenCurrency.equals(CurrencyCode.USD.rawValue())) {
-      return "US$";
+      return "US$ ";
+    } else {
+      symbol = "US$ ";
+    }
+    return symbol;
+  }
+
+  /** Returns the proper currency symbol for the reward pledge amount **/
+  private String getSymbolForRewardCurrency(final Project project) {
```

Comment:
What's the difference between this method and getSymbolForCurrency. Smells like copy pasta 🍝

Reply:
`getSymbolForRewardCurrency` takes in a project and checks against the project currency and `getSymbolForCurrency` takes in the user's preferred currency then displays the correct symbol. They do the same thing just different parameters.

Reply:
fixed!